src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/boutgifter/beregning/BoutgifterBeregningService.kt

```diff
@@ -90,6 +100,73 @@ class BoutgifterBeregningService(
                 )
             }
 
+    /**
+     * Slår sammen perioder fra forrige og nytt vedtak.
+     * Beholder perioder fra forrige vedtak frem til og med revurder-fra
+     * Bruker reberegnede perioder fra og med revurder-fra dato
+     * Dette gjøres for at vi ikke skal reberegne perioder før revurder-fra datoet
+     * Men vi trenger å reberegne perioder som løper i revurder-fra datoet da en periode kan ha endret utgift
+     */
+    private fun settSammenGamleOgNyePerioder(
+        saksbehandling: Saksbehandling,
+        beregningsresultat: List<BeregningsresultatForLøpendeMåned>,
+        forrigeVedtak: InnvilgelseEllerOpphørBoutgifter,
+    ): BeregningsresultatBoutgifter {
+        val revurderFra = saksbehandling.revurderFra
+        feilHvis(revurderFra == null) { "Behandling=$saksbehandling mangler revurderFra" }
+
+        val forrigeBeregningsresultat = forrigeVedtak.beregningsresultat
+
+        val perioderFraForrigeVedtakSomSkalBeholdes =
+            forrigeBeregningsresultat
+                .perioder
+                .filter { it.grunnlag.fom.sisteDagenILøpendeMåned() < revurderFra }
+                .map { it.markerSomDelAvTidligereUtbetaling(delAvTidligereUtbetaling = true) }
+        val nyePerioder =
+            beregningsresultat
+                .filter { it.grunnlag.fom.sisteDagenILøpendeMåned() >= revurderFra }
+
+        val nyePerioderMedKorrigertUtbetalingsdato =
+            korrigerUtbetalingsdato(
+                nyePerioder = nyePerioder,
+                forrigeBeregningsresultat = forrigeBeregningsresultat,
+            )
+
+        return BeregningsresultatBoutgifter(
+            perioder = perioderFraForrigeVedtakSomSkalBeholdes + nyePerioderMedKorrigertUtbetalingsdato,
+        )
+    }
+
+    private fun korrigerUtbetalingsdato(
```

Comment:
Trenger ikke å korrigere utbetalingsdato for boutgifter, då det alltid settes til den første ukedagen i måneden